test/integration/targets/consul/aliases

```diff
@@ -0,0 +1,3 @@
+shippable/posix/group2
+destructive
+needs/target/setup_openssl
```

Comment:
This is unnecessary, since it will be detected automatically from the meta dependencies.